acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/EthSignerProcessRunner.java

```diff
@@ -217,7 +217,7 @@ private void loadPortsFile() {
       portsProperties.load(fis);
       LOG.info("EthSigner ports: {}", portsProperties);
     } catch (final IOException e) {
-      throw new RuntimeException("Error reading Pantheon ports file", e);
+      throw new RuntimeException("Error reading Web3Provider ports file", e);
```

Comment:
Is the file location part of the IOException? If not, that would be nice to have in here)

Reply:
This should be EthSigner, as this function is actually trying to load the EthSigner ports file

Reply:
yes the file location is part of FileNotFoundException so I think this is fine as is